scripts/flattener.py

```diff
@@ -629,22 +633,15 @@ def set_ensembl(cxg_adata, cxg_adata_raw, redundant, feature_keys):
 		unique_to_norm =  set(cxg_adata.var.index.to_list()).difference(set(cxg_adata_raw.var.index.to_list()))
 		if len(unique_to_norm) > 0:
 			print("WARNING: normalized matrix contains Ensembl IDs not in raw: {}".format(unique_to_norm))
+	del cxg_adata_raw.var['feature_biotype'] # remove feature_biotype from var and raw.var
+	del cxg_adata.var['feature_biotype']
```

Comment:
can we prevent these (`feature_biotype`) from being added in the first place?

Reply:
Yes, I want to clean up feature_biotype, but I was thinking to do it in a separate ticket to break up the work. I can make another ticket for that and also move more code from main() to separate functions.

Reply:
works for me